src/pages/DashboardPage.js

```diff
@@ -52,54 +75,141 @@ const DashboardPage = () => {
         ),
       );
 
-      // group daily liquidity data by date
       setFormattedLiquidityData(formattedData.map(({ time, tvl }) => ({ time, ...tvl })));
-      // group daily volume data by date
       setFormattedVolumeData(formattedData.map(({ time, volume }) => ({ time, ...volume })));
     }
   }, [chartData]);
 
+  useEffect(() => {
+    if (comulativeData && Object.keys(comulativeData).length > 0) {
+      const totalTradesPerNetwork = [];
+      const totalVolumePerNetwork = [];
+
+      Object.keys(comulativeData)
+        // include only network specific data
+        .filter((key) => Object.values(SupportedNetwork).includes(key))
+        .forEach((network) => {
+          totalTradesPerNetwork.push({
+            network,
+            value: formattedNum(comulativeData[network].totalTrades),
+          });
+          totalVolumePerNetwork.push({
+            network,
+            value: `$ ${formattedNum(comulativeData[network].totalVolume)}`,
+          });
```

Comment:
```suggestion
          totalTradeAndVolumesPerNetwork.push({
            network,
             volume: formattedNum(comulativeData[network].totalVolume),
            trade: formattedNum(comulativeData[network].totalTrades),
          });
       
```

Reply:
I think it's better to accumulate them in 2 different arrays and then set them in a single state as you suggested, something like:
```js
const totalTradesPerNetwork = [];
const totalVolumePerNetwork = [];

Object.keys(comulativeData)
  // include only network specific data
  .filter((key) => Object.values(SupportedNetwork).includes(key))
  .forEach((network) => {
    totalTradesPerNetwork.push({
      network,
      value: formattedNum(comulativeData[network].totalTrades),
    });
    totalVolumePerNetwork.push({
      network,
      value: `$ ${formattedNum(comulativeData[network].totalVolume)}`,
    });
  });

setFormattedComulativeData({ trades: totalTradesPerNetwork, volume: totalVolumePerNetwork })
```
So in the components it's simpler to pass the required values:
```jsx
<ComulativeNetworkDataCard
  title={'Total trades'}
  icon={<FileText size={22} color={'#50dfb6'} />}
  comulativeValue={formattedNum(comulativeData.totalTrades)}
  networksValues={formattedComulativeData.trades}
/>
```
```jsx
<ComulativeNetworkDataCard
  title={'All time volume'}
  icon={<DollarSign size={22} color={'#50dfb6'} />}
  comulativeValue={`$ ${formattedNum(comulativeData.totalVolume)}`}
  networksValues={formattedComulativeData.volume}
/>
```